lib/rubycritic/cli/options.rb

```diff
@@ -19,6 +19,16 @@ def parse
             @root = path
           end
 
+          opts.on('-b', '--branch BASE_BRANCH,FEATURE_BRANCH', 'Set branches to compare') do |branches|
+            self.base_branch, self.feature_branch = branches.split(',').map { |branch| String(branch) }
+            self.mode = :compare_branches
+          end
+
+          opts.on('-t', '--threshold-score [THRESHOLD_SCORE]',
+                  'Set a threshold for score difference between two branches (works only with -b)') do |threshold_score|
+            self.threshold_score = Integer(threshold_score)
```

Comment:
Build will fail if the decrease in score is more than this value. These were inspired from `Coveralls` settings:

![screen shot 2016-12-15 at 10 49 19 pm](https://cloud.githubusercontent.com/assets/980783/21234480/d22a9c52-c318-11e6-8c90-19cea5cb587d.png)

rubycritic already had `--minimum-score`, so we added the decrease threshold. 
Renamed it to `--maximum-decrease` for clarity.